src/Core/Domain/Order/QueryResult/OrderInvoiceAddressForViewing.php

```diff
@@ -121,7 +127,8 @@ public function __construct(
         string $postCode,
         string $phone,
         string $phoneMobile,
-        ?string $vatNumber = null
+        ?string $vatNumber = null,
+        string $dni = ''
```

Comment:
Why not using `null` when dni is not mandatory?

Reply:
I agree the null makes more sense than an empty string :thinking:

Reply:
Is'nt it against @eternoendless feedback above 😅  ?

Reply:
I prefer null too. Discussion is not closed yet I suppose

Reply:
Fixed